src/AzureOperator.ts

```diff
@@ -1,42 +1,50 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-'use strict';
+"use strict";

```

Comment:
we can get rid of declaration of "use strict" if we set "strict: true" in tsconfig.json.

Reply:
Fixed.